pkg/runtime/runtime.go

```diff
@@ -68,56 +116,97 @@ type InfoOptions struct {
 	annotations     map[string]string
 	runtimePolicy   RuntimePolicy
 	podSpecReplicas []podSpecReplica
+	templateSpec    TemplateSpec
 }
 
-type InfoOption func(options *InfoOptions)
+type InfoOption func(options *InfoOptions) error
 
 var defaultOptions = InfoOptions{}
 
 type podSpecReplica struct {
-	replicas int32
-	name     string
-	podSpec  corev1.PodSpec
+	count   int32
```

Comment:
Why do you prefer count over replicas ?

Reply:
`Replica` is reserved in JobSet, and this Count does not correspoinds to JobSet replica.